### PR TITLE
Windows pop up added

### PR DESCRIPTION
**Pull request by hislordshipprof**
-Added window pop up on the see project button
-Onclick of the button it displays the detail page of that particular project
-Added the functionality on desktop screen size
-Added the functionality on mobile size

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -382,16 +241,18 @@ <h1 class="intro-text">Contact me</h1>
...
+        </div>
       </form>
       </div>
     </footer>
   </main>
-  
+
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
   <script>
     $(document).ready(function () {
```
Comment (leonardodiasb) [inline index.html]: - [ ] Kindly put all your JavaScript codes in JS files, not in the `script` tag. Use JS best practices.

File: projects.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+          <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+        </svg>
+        ${projects[i].jobDescription.year}
+      </li>
+    </ul>
+    <p>${projects[i].description.featured}</p>
+    <ul class="technologies-used">
+      <li>${projects[i].languages.featured[0]}</li>
+      <li>${projects[i].languages.featured[1]}</li>
+      <li>${projects[i].languages.featured[2]}</li>
```
Comment (leonardodiasb) [inline projects.js:120]: - [OPTIONAL] You could use a forEach loop to keep your code DRY in this list.

File: projects.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+    <div class="popup-content">
+      <p>${projects[position].description.detailed}</p>
+      <div>
+        <ul class="technologies-used">
+          <li>${projects[position].languages.detailed[0]}</li>
+          <li>${projects[position].languages.detailed[1]}</li>
+          <li>${projects[position].languages.detailed[2]}</li>
+          <li>${projects[position].languages.detailed[3]}</li>
+          <li>${projects[position].languages.detailed[4]}</li>
+          <li>${projects[position].languages.detailed[5]}</li>
```
Comment (leonardodiasb) [inline projects.js:173]: - [OPTIONAL] You could use a forEach loop to keep your code DRY in this list.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @hislordshipprof , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️
Your JS code looks great ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (hislordshipprof) [reply index.html]: actually all were inside  the scripts .js this cant be outside the html base on how i wrote it that is why its

File: index.html [hunk 4]
```diff
@@ -6,25 +6,27 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <script src="script.js" defer></script>
+  <script src="projects.js" defer></script>
+  <!-- <script src="form.js" defer></script> -->
```
Comment (youmari) [inline index.html]: - [ ] please remove the unused uncommented code in order to keep  you file clean and pro

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @hislordshipprof , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Github flow 
✔️ Great title and short summary
✔️ Good commit messages

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @hislordshipprof , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**